client/lib/plans/features-list.js

```diff
@@ -1010,7 +1010,7 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_PREMIUM_CONTENT_BLOCK ]: {
 		getSlug: () => constants.FEATURE_PREMIUM_CONTENT_BLOCK,
-		getTitle: () => i18n.translate( 'Subscriber-only Content' ),
+		getTitle: () => i18n.translate( 'Subscriber-only content' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.